Upstream: no
Reason: we delete the vendor folder but the missing license breaks the build
  https://github.com/pypa/flit/tree/main/flit_core/flit_core/vendor

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -13,7 +13,7 @@ dependencies = []
 requires-python = '>=3.6'
 readme = "README.rst"
 license = "BSD-3-Clause"
-license-files = ["LICENSE*", "flit_core/vendor/**/LICENSE*"]
+license-files = ["LICENSE*"]
 classifiers = [
     "Topic :: Software Development :: Libraries :: Python Modules",
 ]
